packages/nx/src/command-line/show/command-object.ts

```diff
@@ -63,6 +63,12 @@ const showProjectsCommand: CommandModule<NxShowArgs, ShowProjectsOptions> = {
         type: 'boolean',
         description: 'Show only affected projects',
       })
+      .option('exclude', {
+        type: 'string',
+        alias: ['e'],
+        description: 'Exclude projects that match a given pattern.',
+        coerce: parseCSV,
+      })
```

Comment:
This is under `nx show projects --help` and should not be added here. If the wording is unclear, it should be updated here: https://github.com/nrwl/nx/blob/master/packages/nx/src/command-line/yargs-utils/shared-options.ts#L5

Also, please do not add an alias. It will possibly conflict with aliases in other executors.

Reply:
Good morning (at least in my country) @FrozenPandaz. Removed `exclude` option with your review. Thank you!